source/features/submission-via-ctrl-enter-everywhere.tsx

```diff
@@ -3,7 +3,7 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '../feature-manager';
 
-export default function addQuickSubmit(): void {
+function addQuickSubmit(): void {
```

Comment:
The fun part was that this part of the feature was being exported from `submission-via-ctrl-enter-everywhere`, duplicating it entirely